README.md

```diff
@@ -207,6 +208,15 @@ jobs:
       - uses: actions/checkout@v2
 ```
 
+## Fetch pull request base-ref in addition to merge commit (for diffing)
+
+```yml
+- uses: actions/checkout@v2
+- name: Fetch base_ref HEAD
+  run: git fetch --depth=1 origin +refs/heads/${{github.base_ref}}:refs/remotes/origin/${{github.base_ref}}
```

Comment:
I think this could be simplified to 
```
git fetch --depth=1 origin +${{github.base_ref}}
```
or
```
git fetch --depth=1 origin +refs/heads/${{github.base_ref}}
```

since the destination is the conventional remote ref.